profile/README.md

```diff
@@ -1,11 +1,14 @@
-# 👋 Welcome
+# 🕊️ Free to share code 👩🏽‍💻 Code to share freedom
 
-Dyne.org is a non-profit **free software foundry** and a merry **hacker community**.
+🖖 Welcome, fellow dyne! 
 
-We have more than **20 years of expertise** putting computers at work to **empower people**.
+[Dyne.org](https://dyne.org) is a non-profit **free software foundry** and a merry **hacker community**.<br>We have nearly **30 years of expertise** putting computers at work to **empower people**.
```

Comment:
avoid the <br/> if possibile you can achive that with two newlines in markdown

Reply:
wells potted, seems good also to keep on same line

Reply:
2 news lines creates 2 new lines :/ (why, mr Markdown? Please explain yourself!)  
Github markdown uses <br> wildly and widely. But sure, 2 lines it is!